guides/stable/3D_interactivity.md

```diff
@@ -0,0 +1,131 @@
+# 3D interactivity
+
+## Coordinate systems in napari
+In napari, there are three main coordinate systems: (1) canvas, (2) world, and (3) layer. The canvas coordinates system is the 2D coordinate system of the canvas on which the scene is rendered. World coordinates are the nD coordinates of the entire scene. As the name suggests, the layer coordinates is the nD coordinate system of the data in a given layer. Layer coordinates are specific the each layer's data and are related to the world coordinate system via the layer transforms.
+
+![coordinate-systems](images/3d_interaction_coordianates.png)
+
+## In 3D mode, clicks are lines
+Since the 3D scene is rendered on a 2D surface (your screen), your mouse click does not map to a specific point in space. Thus, when rendering a scene in 3D, napari interprets your click as a line in the direction of the camera view that goes through the point that view ray intersects the camera plane.
```

Comment:
```suggestion
Since the 3D scene is rendered on a 2D surface (your screen), your mouse click does not map to a specific point in space. As the view is a [parallel projection](https://en.wikipedia.org/wiki/Parallel_projection), napari can determine a line through 3D space that intersects the canvas where the user clicked.

Reply:
Good point. I'll add a note about that.